bin/submit_build.py

```diff
@@ -100,7 +100,6 @@ def main():
         "dry-run": ("Do not fetch/install, set debug log level", None, "store_true", False, 'D'),
         "skip-fetch": ("Do not fetch the sources, fail if they are missing", None, "store_true", False, 'n'),
         "bwrap": ("Reinstall via new namespace with bwrap", None, "store_true", False, 'b'),
-        "skip-lmod-cache": ("Do not run Lmod cache after installation", None, "store_true", False, 's'),
```

Comment:
you can keep this and control it with the environment variable? Only if it's useful of course.